runelite-api/src/main/java/net/runelite/api/events/ChatMessage.java

```diff
@@ -39,6 +40,7 @@
  */
 @Data
 @AllArgsConstructor
+@NoArgsConstructor
```

Comment:
As future improvement, this should be immutable, e.g `@Value`, with builder maybe? Because changing stuff on this is not working and can be confusing for people.